webapp/components/suggestion/suggestion_box.jsx

```diff
@@ -24,6 +24,7 @@ export default class SuggestionBox extends React.Component {
         this.handleCompleteWord = this.handleCompleteWord.bind(this);
         this.handleChange = this.handleChange.bind(this);
         this.handleKeyDown = this.handleKeyDown.bind(this);
+        this.handleBlur = this.handleBlur.bind(this);
```

Comment:
The changes to `SuggestionBox` shouldn't actually be necessary. The only reason those other handlers are explicitly passed to the inputs is because of the extra handling that `SuggestionBox` does. Any unrecognized props (like onBlur would be in this case) are automatically passed along